packages/report-server/src/reportBuilder/functions/math.ts

```diff
@@ -3,12 +3,41 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import { sum as mathjsSum, Matrix, filter } from 'mathjs';
+import { sum as mathjsSum, divide as mathjsDivide } from 'mathjs';
 
-export const sum = (numbers?: Matrix): number | undefined => {
-  if (numbers === undefined) {
+const isDefined = <T>(value: T): value is Exclude<T, undefined> => value !== undefined;
```

Comment:
Similar here, but the trade off is that, it will be slightly easier to understand the rest of the codes with them here. Either way is fine to me, I don't want to be picky so I leave it to you to decide.

Reply:
See my point 'Regarding b)' in this comment: https://github.com/beyondessential/tupaia/pull/2624#discussion_r644391666